ui/webui/resources/BUILD.gn

```diff
@@ -64,11 +64,11 @@ if (!is_android && !is_ios) {
         "br_elements/br_shared_style.css",
         "br_elements/br_shared_vars.css",
         "page_specific/settings/br_settings_shared.css",
+        "br_elements/br_toolbar/br_toolbar.css",
+        "br_elements/br_toolbar/br_toolbar_search_field.css",
```

Comment:
Hmm, how come alphabetical order is not enforced in `non_web_component_files` and `css_files`?

Reply:
No idea, but good call!